carbon-design-system/carbon
# chore(project): update to Node v16.13

Closes #9790 

This PR updates our project from v14 LTS to v16 LTS. A notable change is that the `fibers` package is no longer supported in v16 of Node.js. This impacts the `sass` package and its async compilation times.

#### Changelog

**New**

**Changed**

- Update CI files to use v16
- Update nvmrc file to use v16

**Removed**

- `fibers` is removed as a dependency for `carbon-components`

#### Testing / Reviewing

- Verify CI passes as expected and are running on Node v16
- Pull down locally, verify environment is stable for development (run storybook, tests, prettier, eslint, etc)
- Verify no CI workflows reference Node v14 
- Verify Netlify builds are running with Node v16
